docs/api/cookies.md

```diff
@@ -104,3 +104,7 @@ on complete.
 
 Removes the cookies matching `url` and `name`, `callback` will called with
 `callback()` on complete.
+
+#### `cookies.flushStore()`
```

Comment:
It looks like this method takes a callback in the code, assuming that is called on completion.  This should probably be documented here as a parameter 👍